src/org/ods/orchestration/util/MROPipelineUtil.groovy

```diff
@@ -85,34 +90,32 @@ class MROPipelineUtil extends PipelineUtil {
         }
     }
 
-    private void executeODSComponent(Map repo, String baseDir) {
-        this.steps.dir(baseDir) {
-            if (repo.data.openshift.resurrectedBuild) {
-                logger.info("Repository '${repo.id}' is in sync with OpenShift, no need to rebuild")
-                return
-            }
+    void executeODSComponent(Map repo) {
```

Comment:
NOTE: This method is basically unchanged. The huge diff is only due to the removal of `steps.dir(baseDir) {}`. Switching to the correct working dir is done by the surrounding code now to reduce code duplication.